.github/workflows/test.yml

```diff
@@ -8,9 +8,6 @@ jobs:
         fail-fast: false
         matrix:
           destination: [macOS, iOS, tvOS, watchOS]
-          xcode: ["14.1", "13.4.1"]
-    env:
-      DEVELOPER_DIR: /Applications/Xcode_${{ matrix.xcode }}.app/Contents/Developer
```

Comment:
I would change this to fix the version of Xcode at 14.1, so that we are always in control of the version, and not caught out if/when GitHub updates the version on CI.

So we could keep:
```yaml
    env:	
       DEVELOPER_DIR: /Applications/Xcode_14.1.app/Contents/Developer
```